gwcli/gateway.py

```diff
@@ -312,6 +312,12 @@ def clear_config(self, gateway_group):
         # last to ensure we don't fall foul of the api auth check
         gw_list = [gw_name for gw_name in gateway_group
                    if isinstance(gateway_group[gw_name], dict)]
+
+        if this_host() not in gw_list:
```

Comment:
Instead of calling this_host here and multiple times below, let's just call it once

this_gw = this_host()

then replace this_host() calls below with this_gw.